backend/app/services/admin/user_info_service.py

```
@@ -0,0 +1,104 @@
+            return None
+        except Exception as e:
+            logger.error(f"Error getting user profile: {e}")
+            return None
+
+    async def get_user_message_count(self, discord_id: str) -> int:
+        try:
+            supabase = get_supabase_client()
+            res = await supabase.table("message_logs").select(
+                "id", count="exact"
+            ).eq("user_id", discord_id).execute()
+            return res.count or 0
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)